bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```
@@ -274,3 +274,4 @@ config.env.unused.vars = [{0}] unused environment variable
 config.env.variable.name.ambiguity = configurable environment variable ''{0}'' clashes for variable ''{1}'' with \
   variable ''{2}''
 no.worker.message.received = no message received from worker ''{0}'' to worker ''{1}''
+invalid.member.size = insufficient member size: expected at least ''{0}'', found ''{1}''
```

Add a new line at the end of file.
```suggestion
invalid.member.size = insufficient member size: expected at least ''{0}'', found ''{1}''

```

---

Got it.